lib/BaseLogger.js

```diff
@@ -56,6 +56,12 @@ class BaseLogger {
   }
 
   _writeLog(level, message) {
+    // We may have sent an Error;
+    // if we have, the stack is most useful.
+    if (message && message.stack) {
+      return this._writeLog(level, message.stack);
```

Comment:
The first line of `(new Error("message")).stack` is `Error: message`, so we get the error message in the stack.